Lib/test/test_argparse.py

```diff
@@ -1744,27 +1744,44 @@ def convert_arg_line_to_args(self, arg_line):
 # Type conversion tests
 # =====================
 
+def FileType(*args, **kwargs):
+    with warnings.catch_warnings():
+        warnings.filterwarnings('ignore', 'FileType is deprecated',
+                                PendingDeprecationWarning, __name__)
+        return argparse.FileType(*args, **kwargs)
+
+
+class TestFileTypeDeprecation(TestCase):
+
+    def test(self):
+        parser = argparse.ArgumentParser()
```

Comment:
Is this line required? It seems it isn't being used, unless there's some global state I'm forgetting about.

Reply:
Good catch. It is absolutely not needed.
```suggestion
```